web-registry/src/pages/Projects/hooks/useProjectsSellOrders.tsx

```diff
@@ -38,8 +38,6 @@ export const useProjectsSellOrders = ({
   const [loading, setLoading] = useState<boolean>(true);
 
   useEffect(() => {
-    if (projectsWithOrderData.length > 0) return;
```

Comment:
@victordelval This was in a recent change. Do you remember the purpose of this check? I found that it was causing an infinite rendering loop.

Reply:
@mhagel I just added this check to avoid a request loop that I can reproduce with the credit classes present in redwood.

Reply:
OK, I think we will have to find another way to solve that because this creates a circular dependency in the useEffect

Reply:
using a ref broke this out of the render cycle.